website/docs/components/code-block/partials/code/how-to-use.md

```diff
@@ -0,0 +1,148 @@
+## How to use this component
+
+The basic invocation requires `@value` to be passed.
```

Comment:
Should we also mention here how the `value` needs to be provided (they may not read the Component API specs)

Reply:
Addressed in 4773bfa1dcfb5066d7c5a2e15478374f55811e6e